### ISSUE TITLE
Reconsider :global node_modules imports?

### ISSUE THREAD

**Issue by danawoodman** [postcss/postcss-import#350]
I see in issue #179 that no plans were added to allow importing of node_modules styles, but I can import them using the `../node_modules` syntax. Is this something that you would consider revisiting? It's unfortunate to have to use such a clunky system for importing vendor libraries (feels like I'm going back in time!). Thanks!

**Comment by RyanZim** [comment]
@danawoodman We do support importing from `node_modules` with `@import "modulename";`. We just don't support having import statements inside a `:global` block.

**Comment by danawoodman** [comment]
@RyanZim yeah, I understand that. I am asking if you would consider changing that since the behavior is confusing and inconsistent?

**Comment by RyanZim** [comment]
I don't see how it's confusing and inconsistent. The README clearly states:

> This plugin attempts to follow the CSS @import spec; @import statements must precede all other statements (besides @charset).

In short, `@imports` have to be at the top. If their in a `:global` block (or any other block), they're not at the top. The reason they have to be at the top is so we can do proper deduping.

**Comment by danawoodman** [comment]
Let me rephrase; it is confusing and inconsistent compared to the expectation of a new user, especially coming from something like Sass or LESS. That said, what is the recommended approach to do this, use dotpath like I show above?

**Comment by RyanZim** [comment]
A dotpath isn't gonna work inside a `:global` block either.

**Comment by danawoodman** [comment]
Seems to be working just fine for me using:

```css
:global {
  @import ‘../../node_modules/somelib/some.css’;
}
```

**Comment by RyanZim** [comment]
@danawoodman What import plugin are you using? Because that is not working at all for me (and shouldn't work with postcss-import).

**Comment by RyanZim** [comment]
Webpack is almost certainly doing the work here.

**Comment by danawoodman** [comment]
Makes much more sense now. So, in this situation do I even need postcss-import? 

Thanks for working through this with me

**Comment by RyanZim** [comment]
If you're using webpack, you probably don't want postcss-import.

**Comment by danawoodman** [comment]
Rodger that, thanks for clarifying
